work-item-comments.go

```diff
@@ -41,12 +42,12 @@ func (c *WorkItemCommentsController) Create(ctx *app.CreateWorkItemCommentsConte
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
 		}
-
 		reqComment := ctx.Payload.Data
-
+		markup := rendering.NilSafeGetMarkup(reqComment.Attributes.Markup)
 		newComment := comment.Comment{
 			ParentID:  ctx.ID,
 			Body:      reqComment.Attributes.Body,
+			Markup:    markup,
```

Comment:
this will be addressed in https://github.com/almighty/almighty-core/issues/724, actually